flutter_appauth_platform_interface/lib/src/common_request_details.dart

```diff
@@ -26,6 +26,9 @@ class CommonRequestDetails implements Mappable {
   /// Whether to allow non-HTTPS endpoints (only applicable on Android)
   bool allowInsecureConnections;
 
+  /// Whether to use an ephemeral session and not share the cookies across the OS
```

Comment:
Can you add a comment to indicate that it's specific to iOS? This can be noted in a separate line that is separate by a blank line to follow the [effective Dart documentation guidelines](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph)